static/sass/_snapcraft_custom-cols.scss

```diff
@@ -0,0 +1,17 @@
+@mixin snapcraft-custom-cols {
+  .col-no-scale {
+    @media (min-width: 772px) and (max-width: 1083px) {
```

Comment:
Where do these numbers come from? Are they somehow related to breakpoints from vanilla? At least the 772px which seems to be repeated? Would be nice to have those taken from some variables if possible.

I assume the max-widths come from trial and error (just looking at when we want items to drop, or what minimal size we want them to have). Because of that (and their quite arbitrary nature) I'm not really sure how to name those classes. `col-no-scale` doesn't say anything. And it's not like it disabling scaling. It's just it changes the breakpoints of when grid changes. I kind of prefer the `col-land-select` one, as it specifically says for what this col class was made, so maybe we could stick to that?
I guess `col-no-scale` is used in different places and that's why the name is so vague? If it's mostly for logos, maybe call it `col-logo`?